widget/embedded/src/types/config.ts

```diff
@@ -145,6 +145,11 @@ export type Features = Partial<
 > &
   Partial<Record<'experimentalRoute', 'disabled' | 'enabled'>>;
 
+export type Manifest = {
```

Comment:
The name is a general name and doesn't show anything that it relates to Trezor only.